packages/esm-patient-chart-app/src/root.component.tsx

```diff
@@ -10,7 +10,6 @@ export default function Root() {
     <BrowserRouter basename={spaRoot}>
       <SideMenu />
       <Route path={dashboardPath} component={PatientChart} />
-      <Route path={basePath} component={ContextWorkspace} />
```

Comment:
I don't agree with this change.

The context for the workspace should work globally in patient chart and not be confined to the patient chart component.